packages/swapkit/wallets/package.json

```diff
@@ -8,6 +8,7 @@
     "@swapkit/wallet-keystore": "workspace:*",
     "@swapkit/wallet-ledger": "workspace:*",
     "@swapkit/wallet-okx": "workspace:*",
+    "@swapkit/wallet-polkadot.js": "workspace:*",
```

Comment:
I would name it without the '.' so just polkadotjs

Reply:
done

Reply:
done